test/Masa.Contrib.Data.Contracts.EF.Tests/Domain/Entities/Students.cs

```diff
@@ -13,4 +13,9 @@ public Students()
     public int Age { get; set; }
 
     public DateTime RegisterTime { get; private set; }
+
+    public void ChangeEnableState(bool state)
```

Comment:
Bad UT. Why change enable state set IsDeleted?

Reply:
This is just to reflect that the deleted state can be queried again. It has no special meaning and does not encourage users to restore data by changing the state. It has been modified here.